### ISSUE TITLE
Citation of waveglow_256channels_v5

### ISSUE THREAD

**Issue by philippaichinger** [NVIDIA/waveglow#226]
Hi,
is there a recommended way of citing published model parameters? I am using waveglow_256channels_v5. Also, I would be interested in some hard facts regarding this model parameters (what training data, how long trained etc.)
Best, Philipp

**Comment by rafaelvalle** [comment]
You can cite the paper and the GitHub repo or the Mellotron paper which describes this waveglow model.

v5 was trained on a studio quality single female speaker dataset with about 20h. We've trained the model for about 1.5M iterations.

**Comment by philippaichinger** [comment]
OK thanks. Is it really 1.5 mio. iterations (within which 1 batch of a few training samples is used), or epochs (within which the network sees each training sample once, i.e., all batches are used once)?

**Comment by rafaelvalle** [comment]
1.5M iterations

**Comment by anderleich** [comment]
Hi,

How did you decide to stop the training process? Is there any metric you used or it was just inference audio quality?
